### PR TITLE
fix(confirmation): fix appearance for dropdown menu actions

### PR DESCRIPTION
**Pull request by andreybakanovsky**

### CODE REVIEW

**Comment by andreybakanovsky** [comment]
image
image

File: app/javascript/menu/dropdowns.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ document.addEventListener('turbolinks:load', () => {
 
   dropdowns.forEach(function (dropdown) {
     dropdown.addEventListener('click', function (e) {
-      e.stopPropagation()
+      // e.stopPropagation()
```
Comment (ka8725) [inline app/javascript/menu/dropdowns.js:10]: Will we need this code? When and in wick circumstances?

**Review by ka8725 — CHANGES_REQUESTED** [review]
That would be great to add more info with description what the fix does

File: app/javascript/menu/dropdowns.js (hunk 1 above)
Comment (andreybakanovsky) [reply app/javascript/menu/dropdowns.js:10]: It's really no need. I'd like to leave as a result of battles with UI(dropdown menu+hiding+speed of scroll). I think it’s gonna be okay

**Review by ka8725 — APPROVED** [review]
Thanks!

**Pull request merged**